### ISSUE TITLE
Add support for Lumen

### ISSUE THREAD

**Issue by spiritix** [spiritix/lada-cache#92]
The library has not yet been tested with Lumen, it seems that there are some issues at least.

**Comment by merlinblack** [comment]
We are using lada-cache in production on Lumen. As far as I can remember we only needed to provide a config_path function.

`<?php

if ( ! function_exists('config_path'))
{
    /**
     * Get the configuration path.
     *
     * @param  string $path
     * @return string
     */
    function config_path($path = '')
    {
        return app()->basePath() . '/config' . ($path ? '/' . $path : $path);
    }
}
`

**Comment by michael-rubel** [comment]
Looks like it's zero point to keep supporting Lumen now, due to the release of Laravel Octane. Lumen users should upgrade applications to Laravel if they wanna keep their apps up with new features and performance boosts.

**Comment by spiritix** [comment]
Yeah I agree, if the only reason for using Lumen was performance, then it's better to upgrade to Laravel anyway. Framework bootstrap only makes a very small part of application loading time, and nowadays with PHP8 and even Octane it's pretty much negligible.
